nextflow_schema.json

```
@@ -292,24 +296,58 @@
             "description": "Options specific for the ExomeDepth execution",
             "default": "",
             "properties": {
+                "yamlconfig": {
+                    "type": "string",
+                    "description": "YAML config file to use in BEDGOVCF module"
+                },
                 "roi_auto": {
                     "type": "string",
                     "description": "Path to the default ROI (regions of interest) BED file of autosomal (merged) regions to be used for CNV analysis",
                     "format": "file-path",
                     "pattern": "^\\S+\\.bed$",
-                    "mimetype": "text/plain",
-                    "default": null
+                    "mimetype": "text/plain"
                 },
                 "roi_chrx": {
                     "type": "string",
                     "description": "Path to the default ROI (regions of interest) BED file of chrX (merged) regions to be used for CNV analysis",
                     "format": "file-path",
                     "pattern": "^\\S+\\.bed$",
-                    "mimetype": "text/plain",
-                    "default": null
+                    "mimetype": "text/plain"
                 }
             },
             "required": ["roi_auto", "roi_chrx"]
+        },
+        "vep_parameters": {
+            "title": "VEP parameters",
+            "type": "object",
+            "description": "",
+            "default": "",
+            "properties": {
+                "vep_cache_version": {
+                    "type": "integer",
+                    "description": "Cache version",
+                    "default": 105
+                },
+                "species": {
+                    "type": "string",
+                    "description": "Species",
+                    "default": "homo_sapiens"
+                },
+                "vep_cache": {
+                    "type": "string",
+                    "description": "Path to the VEP cache"
+                },
```

```suggestion
                "vep_cache": {
                    "type": "string",
                    "format": "file-path",
                    "description": "Path to the VEP cache"
                },
```
You can also add `exists` here if you change the `vep_cache` for your tests to a real directory (doesn't need to be a VEP cache, just a real directory)

---

Added